llvm/include/llvm/ADT/SmallSet.h

```diff
@@ -14,14 +14,13 @@
 #ifndef LLVM_ADT_SMALLSET_H
 #define LLVM_ADT_SMALLSET_H
 
+#include "llvm/ADT/STLExtras.h"
```

Comment:
I understand that this include existed before this patch, but can it be avoided?
Even DenseMap.h doesn't include it.